qase-python-commons/src/qaseio/commons/config.py

```diff
@@ -3,7 +3,13 @@
 
 class ConfigManager:
 
-    def __init__(self, config_file = './qase.config.json', env_vars_prefix = 'QASE_'):
+    def __init__(self, config_file=None, env_vars_prefix = 'QASE_'):
+        if not config_file:
+            try:
+                from lib.util.common import BASE_DIR
```

Comment:
This is done to read qase.config.json located in the root project directory (when cwd is not set as project directory - default behavior in pycharm). This module has a different location in docker, so we can use a reference path like "../../..."
I'm open to suggestions on how to change it not to use an external lib_testbed module

Reply:
I do not see an option either. On the other hand this lib is "so ours" recently due to many changes, so I let's keep it like it is.